.spelling

```diff
@@ -318,6 +318,8 @@ OpenID_Connect
 OpenShift
 OpenSSL
 openssl
+option1
```

Comment:
this is troubling.  Each time we add a link, we don't want to add that to  the .spelling file.

@geeknoid any offer for a solution to this problem?

Reply:
I would suggest changing the label to option-1 to avoid problems with the spellchecker.
```suggestion
```

Reply:
Change to `option-1` and `option-2`